src/controller.financial.js

```diff
@@ -1,6 +1,6 @@
 ﻿'use strict';
 
-import {BarController} from 'chart.js';
+import {BarController, Chart} from 'chart.js';
```

Comment:
maybe this should be `defaults` instead of `Chart` and then down below change `Chart.defaults` to just `defaults`

Reply:
Thanks for raising this as an issue. I just pushed a change to `master` to fix this part